### DISCUSSION TITLE
[Ton Comment limit] when i add comment field for TON, i got empty encode transaction.

### DISCUSSION THREAD

**Discussion by weixuefeng** [trustwallet/wallet-core#3885] [Ideas & Feature request]
when i add comment:
```
data:application/json,{"p":"ton-20","op":"transfer","tick":"mtmt","amt":"10000","to":"UQCCJjwbdw9gXLnVhzcVKVlUx123456789U1"}
```

i got empty transaction encode.

when i add comment:
```
data:application/json,{"p":"ton-20","op":"transfer","tick":"mtmt","amt":"10000","to":"UQCCJjwbdw9gXLnVhzcVKVlUx123456789U"}
```

i got raw transaction encode.

so i want to know, i can i adjust the comment length on TON? 
@satoshiotomakan

**Comment by weixuefeng** [comment]
The test input.writeToBuffer().toHex() is `0x0a2094a504d69536a30617742527a4233726c67346b4f6538626c4775687a634534612cd0108021230455141594a504d69536a30617742527a4233726c67346b4f6538626c4775687a6345346765516e6d54464c675862426618002078280330dc9ffab2063a8801646174613a6170706c69636174696f6e2f6a736f6e2c7b2270223a22746f6e2d3230222c226f70223a227472616e73666572222c227469636b223a226d746d74222c22616d74223a223130303030222c22746f223a2245514459787765444e336576587a59416c30797364506f4d704c36767a665f663347783536723533414b4e766d384b43227d4000`

result is null. with any error and result.

How can i check it?

**Comment by ronaldoguedess** [comment]
I can't set comment over 38 chars... 38 is the limit.. is there a way to increase it? or is it the TON cap?
